news/_posts/2019-09-10-the-review-process.md

```diff
@@ -0,0 +1,89 @@
+- We also check that best practices are implemented into the Pull Request, be it standard conventions or practices like [PSR](https://www.php-fig.org/psr/) or best security recommandations like the [OWASP](https://www.owasp.org/) recommandations. When people use PrestaShop to build a custom website, it is likely that they will follow the practices they see implemented in the Core so we try to think of the code we merge as an example that people will use.
+- PrestaShop has grown huge over the years, both as a codebase and as a software. There are hundred of features built in the software, but it is not often that people use them all. That is why some contributions sometimes need to be improved because they are not compatible with one of the existing features of the software. Common examples are the multi-store mode or the RTL (Right-To-Left) mode, two feature that a lot of developers do not use (because it adresses specific merchant or countries needs) and consequently when they submit a Pull Request, they do not check whether the modification they submit is compatible with these modes.
+- We follow [SemVer](https://semver.org/). This means that, as much as possible, when we release minor and patch versions, we do not introduce breaking compatibility changes. We make sure the Pull Request does not introduce such changes as some developers submit pull requests without being aware of this constraint.
+- We also have a vision of what PrestaShop should evolve to in order to follow the new trends in the software world. Although a big codebase like PrestaShop is very slow to move, we keep in mind the future architecture and features we want to bring and we check whether the Pull Request is following this direction. For example today PrestaShop relies heavily on jQuery for its frontend features, and we started to introduce Vue.js . So if tomorrow we see a Pull Request that is using React.js we might refuse it in order to keep a consistency in the technology stack used for the project.
+
+So as you can see, in a Pull Request we review a lot of things ! To do all of these checks, we do not only look at the code on github. We sometimes pull the branch on our own computer to test and navigate the code in order to make sure all of these requirements are met. We discuss with team members about parts we are not sure. We can even sometimes run benchmarks or audits for critical pieces of code.
+
+Some Pull Requests are very easy to review because they are related to a part of the code we know very well, other instead are very hard to review because they are related to a complex topic, a complex area of the code or have a huge global impact on the software that is very hard to estimate and assess. These last reviews can take hours or days as we try our very best to make sure every contribution merged in the project meets the level of quality we want for it.
```

Comment:
```suggestion
Some Pull Requests are very easy to review because they are related to a part of the code we know very well, other instead are very hard to review because they are related to a complex topic, a complex area of the code or have a huge global impact on the software that is very hard to estimate and assess. Reviews can take hours or days as we try our very best to make sure every contribution merged in the project meets the level of quality we want for it.
```